Levels/NewStarbase/NewStarbase.prefab

```diff
@@ -4383,8 +4376,9 @@
                     "Id": 8615741586168280576,
                     "m_template": {
                         "$type": "MultiplayerSample::HUDComponent",
-                        "RoundNumberId": 8,
-                        "RoundTimerId": 4
+                        "RoundNumberId": "",
+                        "RoundTimerId": "",
+                        "RoundSecondsRemaining": ""
```

Comment:
Is this change intentional? If so, what does it fix? If not, does the HUD still work?

Reply:
No this was not something I changed and no, the HUD doesn't work. I know it worked in testing earlier, so i'm not sure what happened.